trie/trieBatchManager/trieBatchManager_test.go

```diff
@@ -0,0 +1,131 @@
+package trieBatchManager
+
+import (
+	"testing"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTrieBatchManager(t *testing.T) {
+	t.Parallel()
+
+	tbm := NewTrieBatchManager()
+	assert.False(t, check.IfNil(tbm))
+
+	assert.False(t, tbm.isUpdateInProgress)
+	assert.False(t, check.IfNil(tbm.currentBatch))
+	assert.True(t, check.IfNil(tbm.tempBatch))
+}
+
+func TestTrieBatchManager_TrieUpdateInProgress(t *testing.T) {
+	t.Parallel()
+
+	tbm := NewTrieBatchManager()
+	tbm.currentBatch.Add([]byte("key1"), core.TrieData{})
```

Comment:
```suggestion
	tbm.Add([]byte("key1"), core.TrieData{})
```
use Add method directly? and in all other places

Reply:
Done.